cmd/wrestic/main.go

```diff
@@ -44,13 +44,16 @@ var (
 	restoreType   = flag.String("restoreType", "", "Type of this restore, folder or S3")
 	restoreFilter = flag.String("restoreFilter", "", "Simple filter to define what should get restored. For example the PVC name")
 	archive       = flag.Bool("archive", false, "")
-	stdinOpts     arrayOpts
+	tags          arrayOpts
```

Comment:
Please fix the `String()` function of the `arrayOpts` type

Reply:
This actually gives me an idea on how to improve another part. Thanks